webapp/src/components/emptyCenterPanel.tsx

```diff
@@ -1,34 +1,170 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-import React from 'react'
-import {FormattedMessage} from 'react-intl'
+import React, {useCallback, useEffect} from 'react'
+import {FormattedMessage, useIntl} from 'react-intl'
+import {generatePath, useHistory, useRouteMatch} from 'react-router-dom'
 
 import {getCurrentWorkspace} from '../store/workspace'
-import {useAppSelector} from '../store/hooks'
+import {useAppSelector, useAppDispatch} from '../store/hooks'
+import {Utils} from '../utils'
+import {Board} from '../blocks/board'
+import {getGlobalTemplates, fetchGlobalTemplates} from '../store/globalTemplates'
+import {getSortedTemplates} from '../store/boards'
+import AddIcon from '../widgets/icons/add'
+import BoardIcon from '../widgets/icons/board'
+import octoClient from '../octoClient'
+
+import {addBoardTemplateClicked, addBoardClicked} from './sidebar/sidebarAddBoardMenu'
+import {addBoardFromTemplate, BoardTemplateMenuItemOptionsButton} from './sidebar/boardTemplateMenuItem'
+
 import './emptyCenterPanel.scss'
 
-const EmptyCenterPanel = React.memo(() => {
-    const workspace = useAppSelector(getCurrentWorkspace)
+type ButtonProps = {
+    buttonIcon: string | React.ReactNode,
+    title: string,
+    readonly: boolean,
+    onClick: () => void,
+    showBoard?: (boardId: string) => void
+    boardTemplate?: Board
+}
+
+const EmptyCenterPanelButton = React.memo((props: ButtonProps) => {
+    const {onClick, buttonIcon, title, readonly, showBoard, boardTemplate} = props
 
     return (
-        <div className='EmptyCenterPanel'>
-            {workspace && workspace.id !== '0' &&
-            <div className='WorkspaceInfo'>
-                <FormattedMessage
-                    id='EmptyCenterPanel.workspace'
-                    defaultMessage='This is the workspace for:'
+        <div
+            onClick={onClick}
+            className='button'
+        >
+            <i>{buttonIcon}</i>
```

Comment:
This is sheering the emojis-
<img width="838" alt="Screenshot 2021-10-04 at 1 25 43 PM" src="https://user-images.githubusercontent.com/18575143/135814276-0f9972c2-ca7f-4a30-8b38-61d7d095580f.png">

We should use a span here.

Reply:
Oooooh Nice! Much better 😄